tests/unit/LedgerLivePlatformSDK/index.spec.ts

```diff
@@ -224,9 +225,14 @@ describe("LedgerLivePlatformSDK/index.ts", () => {
         }) as ChaiSpies.Spy;
 
         const res = await SDK.listAccounts();
+
+        console.log({
+          res,
+          dummyAccountList,
+        });
         expect(res).to.deep.eq(dummyAccountList);
         expect(spy).to.be.have.been.called.with(
-          '{"jsonrpc":"2.0","method":"account.list","id":1}'
+          '{"jsonrpc":"2.0","method":"account.list","params":{},"id":1}'
```

Comment:
You got a test for adding the new params for including ERC20?

Reply:
Nope, but what are we testing here because including or not ERC20 is implemented in the ledger live side. The SDK is just serving "whatever is provided to it"

Reply:
Yeah so could we not have a test that uses a dummy list of accounts (like the `dummyCurrencyList` below) and make sure that the expected filter matches reality depending on if we pass the `includeTokens?` param in?

Reply:
we could but then you are testing the server's aptitude to answer your query. The client simply ask stuff with parameters and output what was returned by the server dumbly. There are no logic to test beside "the parameters are effectively sent to the server"